tests/console/openvpn.pm

```diff
@@ -0,0 +1,65 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Setup and test point-to-point VPN using static key with OpenVPN
+# Maintainer: Dehai Kong <dhkong@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use lockapi;
+use mmapi;
+use mm_network;
+use mm_tests;
+
+sub run {
+
+    my $password = $testapi::password;
+    select_console "root-console";
+
+    # Set up openvpn client's IP address
+    configure_static_network('10.0.2.10/24');
+
+    # Stop firewall
+    assert_script_run("SuSEfirewall2 stop");
```

Comment:
Why do you disable the firewall? We don't want users to disable the firewall to make use of a VPN.

Reply:
first thought in my mind , it's the firewall will impact openvpn work. So , I disabled it . in fact , the firewall will be open. I will modify this pm . thanks!!